tests/foreman/ui/test_host.py

```diff
@@ -172,13 +173,18 @@ def setUpClass(cls):
         resource_url = u'qemu+ssh://root@{0}/system'.format(
             settings.compute_resources.libvirt_hostname
         )
-        cls.computeresource = entities.LibvirtComputeResource(
-            url=resource_url
-        ).search()
-        if len(cls.computeresource) >= 1:
-            cls.computeresource = cls.computeresource[0]
-            cls.computeresource.location = [cls.loc]
-            cls.computeresource.organization = [cls.org]
+        comp_res = entities.LibvirtComputeResource().search()
+        # Exclude everything but Libvirt compute resources from search result
+        if len(comp_res) >= 1:
```

Comment:
This if is not needed, doing the list comprehension will return an empty list if comp_res is also empty.